Tests/PropertyTests.fs

```diff
@@ -0,0 +1,145 @@
+module RandomTests
+
+open Xunit
+open FsCheck
+open FsCheck.Xunit
+open FsCheck.Arb
+
+open MyListSorts.SortFunctions
+open MyListSorts.TypeConvertFunctions
+
+type Overrides() =
+    static member Float() =
+        Arb.Default.Float()
+        |> filter (fun f -> not <| System.Double.IsNaN(f) && not <| System.Double.IsInfinity(f))
+
+Arb.register<Overrides> () |> ignore
+
+
+[<Properties(MaxTest = 10)>]
+type RandomTestsForBubleSort() =
+
+    [<Property>]
+    member _.IntTest(test_cases: int list) =
+        let actual = bubble_sort (ConvertListToMyList test_cases) |> ConvertMyListToList
+        let expected = List.sort test_cases
+        Assert.True(List.forall2 (=) actual expected)
```

Comment:
Can we use [`Assert.Equeal`](https://xunit.net/docs/hash-sets-vs-linear-containers) here?